### DISCUSSION TITLE
Option to search by year

### DISCUSSION THREAD

**Discussion by gg-gg1** [deckerst/aves#342] [Ideas]
Hi! Once again, I must say that I enjoy the app and want to thank you @deckerst for your amazing work. May I just suggest something?
On search, it shows on this day and months options. Can you add option for years? I appologise if there is already and I didn`t see it.

Thanks.

**Comment by deckerst** [comment]
There are a couple of things you could try:
- in the `Stats` page, there's a graph by date. It shows years or months or days, depending on the collection. If you tap on the graph, it will show a filter. You can play with that filter like any other.
- in the `Search` page, you can type `year<2011`, `year=2020`, etc.

**Reply by gg-gg1** [reply]
Yes, it works. I will use the second option for when I need a specific year.  Thanks.
